Polished Project Info

vabene1111 opened the pull request:
Reason for PullRequest
======================
While waiting for Arma to finish downloading, i had time to work one some DOC.
More will follow as soon as i explore alle the Features.

Description of the Changes
==========================
Change Readme, PR Template, Issue Template and Contribution Info to look a little more polished.

Notes
=====
Just did this while waiting, if you dont like it just reject/close.

vabene1111 committed:
Updated Project Info

Updated Project Readme, Issue/PR Templates
Added short instructions on how to setup Project in VS

vabene1111 committed:
Fixed Link

accidentally linked to fork instead of origin repository

trdwll commented on CONTRIBUTING.md:
```diff
@@ -1,3 +1,17 @@
+
+To start working on ArmA.Studio
+
+1. If you have not already, [download](https://www.visualstudio.com/de/downloads/) and install Visual Studio
```
Should change the url to https://www.visualstudio.com/downloads/ rather than /de/

trdwll replied:
Forgot to mention that the /de/ link redirects to /downloads/, so this isn't necessary. However it may confuse people if they get redirected to a /de/ link if they're not german. Also /downloads/ is the default so it's rather pointless to redirect the user.

vabene1111 replied:
ups, did't see the /de/

dedmen commented on README.md:
```diff
@@ -1 +1,30 @@
+[Website (Download)](https://x39.io/projects?project=ArmA.Studio)
+
+[Bi-Forum Thread](https://forums.bistudio.com/forums/topic/203020-armastudio-the-open-source-community-made-ide-for-arma/) / [Armaholic](http://www.armaholic.com/page.php?id=32550)
+
```
`[Armaworld (German)](https://armaworld.de/index.php?thread/2838-alpha-arma-studio-die-community-made-opensource-ide-f%C3%BCr-arma/)`

dedmen replied:
Final result looks like
`Bi-Forum Thread / Armaholic Armaworld (German)` Need to add more Newlines if it should be on seperate lines. Or add another `/`

jonpas commented on ISSUE_TEMPLATE.md:
```diff
@@ -1,17 +1,61 @@
 * While step 2: that and that should happen
 
+
+Additional Notes
```
This is all duplicated now.

Should be kept as short as possible with as much information as possible, for easy editing upon making a new issue. And without code blocks, you want most of it parsed in markdown. Additionally I'd avoid `====` but use `### ` at start of line or bold headers.

vabene1111 replied:
Yes i was considering to remove the === as i did't like it myself but did not want to change your template to much :) i will change it

jonpas commented on PULL_REQUEST_TEMPLATE.md:
````diff
@@ -1,12 +1,48 @@
+Please use this Template for all Pull Requests to this repository.
+
+```markdown
+Reason for PullRequest
````
Same as above, duplicated, mind, this file is directly copied to the comment section upon creating a new PR.

dedmen reviewed (changes requested):
Issues pointed out in comments.

vabene1111 committed:
Updated Contributing

Changed /de/ link for universal Link

vabene1111 committed:
Update ISSUE_TEMPLATE.md

As i did't remember the files being pasted directly by github i made it copy friendlier before,
now back to normal with slightly improved markdown

vabene1111 committed:
Update PULL_REQUEST_TEMPLATE.md

As i did't remember the files being pasted directly by github i made it copy friendlier before,
now back to normal with slightly improved markdown

If PR is created trough desktop/terminal client text must be pasted manually, but i guess most people will use the WebInterface anyway

vabene1111 committed:
Update README.md

Added Arma World Link to link list

vabene1111 commented:
All mentioned Issues should be fixed, since i mostly use desktop/terminal clients i did't notice that the template files would be copied automatically, so i made it copy friendlier, but since most people use the Web Interface its better this way.

Also changed === to ## , did't do it before because i thought whoever wrote the files liked === better.

dedmen commented on ISSUE_TEMPLATE.md:
```diff
@@ -1,66 +1,18 @@
+## Additional Notes
 * Relevant information
 * Document title
 * [Some document]([Linked Document](http://LinkToDoc)
```
[Some document]([Linked Document](http://LinkToDoc)
`)` missing
Also I think 
`[Linked Document](http://LinkToDoc)` would be enough

dedmen replied:
[Some document]([Linked Document](http://LinkToDoc)
`)` missing or one `(` too much.

vabene1111 committed:
Update README.md

Added /

vabene1111 committed:
Update ISSUE_TEMPLATE.md

Should not use the web editor ...

vabene1111 commented:
should not have used the web editor ..
sry for all the mistakes, should be fixed now
